Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
Thanks for the explanation, had forgotten about that part, definitely not that simple. I do however feel a lot of that confusion can be contributed to the fact that there is no way to see the merged Fusion configuration anywhere without manually debugging. Currently there's simply no way to get an overview of the final prototypes nor node types from the code.